[docs] Added list of preflight skips flags (#10916)
deckhouse/deckhouse: docs/documentation/pages/installing/README.md

## Installing Deckhouse

{% alert level="info" %}

> When installing the Enterprise Edition of Deckhouse from the official container registry `registry.deckhouse.io`, you must first authenticate using a license key:

Reviewer: ```suggestion
When you install Deckhouse Enterprise Edition from the official container registry `registry.deckhouse.io`, you must first log in with your license key:
```
